RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
+
+This release fixes an issue when trying to use `Annotated[strawberry.auto, ...]`
+on python 3.10 or older, which got unmasked after the fix from 0.196.1.
```

Comment:
can we explain what unmasked is here?

Reply:
Hrm, maybe the "unmasked" term is not the correct one in english?

Basically the issue was there, but was not happening because of the typo that 0.196.1 fixed.

Reply:
like, 0.196.1 made that issue evident